pkgs/tools/backup/stenc/default.nix

```
@@ -1,14 +1,29 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, genericUpdater
+, common-updater-scripts
+, autoreconfHook
+}:
 
 stdenv.mkDerivation rec {
-  version = "1.0.7";
   pname = "stenc";
+  version = "1.0.8";
 
   src = fetchFromGitHub {
     owner = "scsitape";
     repo = "stenc";
     rev = version;
-    sha256 = "1778m1zcyzyf42k5m496yqh0gv6kqhb0sq5983dhky1fccjl905k";
+    sha256 = "0dsmvr1xpwkcd9yawv4c4vna67yag7jb8jcgn2amywz7nkpzmyxd";
+  };
+
+  nativeBuildInputs = [ autoconf automake autoreconfHook ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook ];
```

autoreconfHook should include those.